### ISSUE TITLE
excon 0.77.0 gem is missing files

### ISSUE THREAD

**Issue by elektronaut** [excon/excon#728]
It looks like the published gem is missing the lib and data dirs:

```
~/./v/2/l/r/g/2.7.0/gems $ ll excon-0.77.0 
-rw-r--r--  1 inge  staff   1.2K Oct 16 18:29 CONTRIBUTING.md
-rw-r--r--  1 inge  staff   5.2K Oct 16 18:29 CONTRIBUTORS.md
-rw-r--r--  1 inge  staff   1.1K Oct 16 18:29 LICENSE.md
-rw-r--r--  1 inge  staff    19K Oct 16 18:29 README.md
-rw-r--r--  1 inge  staff   1.7K Oct 16 18:29 excon.gemspec

~/./v/2/l/r/g/2.7.0/gems $ ll excon-0.76.0
total 80
-rw-r--r--  1 inge  staff   1.2K Aug  3 13:26 CONTRIBUTING.md
-rw-r--r--  1 inge  staff   5.2K Aug  3 13:26 CONTRIBUTORS.md
-rw-r--r--  1 inge  staff   1.1K Aug  3 13:26 LICENSE.md
-rw-r--r--  1 inge  staff    19K Aug  3 13:26 README.md
drwxr-xr-x  3 inge  staff    96B Aug  3 13:26 data
-rw-r--r--  1 inge  staff   1.7K Aug  3 13:26 excon.gemspec
drwxr-xr-x  4 inge  staff   128B Aug  3 13:26 lib
```

**Comment by geemus** [comment]
@elektronaut Thanks for the heads up. Not sure how/why things went wrong there (seems something in my scripts doesn't work after OS updates and other things). I've yanked 0.77.0 for now to avoid further confusion for others, will push a new release once I've fixed the issue.

**Comment by geemus** [comment]
Turns out the Gemfile was relying on a bash-specific expansion, which was (unbeknownst to me) silently failing in zsh. I have been playing with zsh for a while, so not sure how/why I didn't run into this previously. In any event, I believe it should now be fixed, and I've released 0.78.0 which should basically be 0.77.0, but with all the files it should have had in the first place.

@elektronaut thanks again for the quick catch and report so that I could get this fixed ASAP.
